0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -1,11 +1,9 @@
 # Interledger Architecture
 
-This document outlines the Interledger architecture and explains how the different layers relate to each other.
+This document outlines the Interledger architecture and protocol suite and explains how the different layers relate to each other. The [Interledger Protocol (ILP)](#interledger-protocol-ilp) is one layer in the Interledger Architecture.
```

Comment:
Saw the same wording in the index. Minor issue, but "architecture and protocol suite" reads weird to me, since the decision to split it into multiple protocol layers is just a feature of the architecture. So it's kinda like saying: "I like this car and sports car." Not really wrong, just a bit strange way to say it.

Like the clarifying sentence though on the relationship between ILP and Interledger architecture.

Would lower case "architecture".